resources/views/content/scripts.twig

```diff
@@ -7,7 +16,11 @@
             action = '/' + App.language + '/search';
         }
 
-        document.location = action + '?query=' + formElement.querySelector('input.search-input').value;
+        if (formElement.querySelector('input.search-input').value.trim() !== '') {
+            handleSearchQuery(formElement, action);
+        } else {
+            handleSmartSuggestClick(formElement, action);
```

Comment:
Do we also need to handle Guided Shoppping with empty query?

Reply:
The Shopping Guide works just fine.